src/runtime/Makefile

```diff
@@ -462,7 +462,7 @@ ifneq (,$(QEMUCMD))
 
     DEFAULTVCPUS_NV = 1
     DEFAULTMEMORY_NV = 2048
-    DEFAULTTIMEOUT_NV = 500
+    DEFAULTTIMEOUT_NV = 1200
```

Comment:
This will also increase the timeout for the non-CC NV case. May or may not be intended/matter, just pointing out.